opencensus/stats/internal/view_data_impl.cc

```diff
@@ -157,6 +168,7 @@ ViewDataImpl::ViewDataImpl(const ViewDataImpl& other)
       aggregation_window_(other.aggregation_window_),
       type_(other.type()),
       start_time_(other.start_time_) {
+  new (&start_times_) DataMap<absl::Time>(other.start_times_);
```

Comment:
Please put this in the initializer list and don't use the placement-new.

Reply:
Done